dumoRecordsHandler/Course/Course.cpp

```diff
@@ -0,0 +1,19 @@
+//
+// Created by emyrmak on 7/23/2019.
+//
+
+#include "Course.h"
+
+Course::Course(int id, std::string name, int teacherId): id(id), name(name), teacherId(teacherId) {
```

Comment:
To avoid confusion in code reader, it's nice to add "_" or "m_" prefix to class atributes. E.g.: _name;